packages/framework/src/Model/Order/Watcher/TransportAndPaymentWatcher.php

```diff
@@ -1,261 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Shopsys\FrameworkBundle\Model\Order\Watcher;
-
-use Shopsys\FrameworkBundle\Model\Order\OrderData;
-use Shopsys\FrameworkBundle\Model\Order\Preview\OrderPreview;
-use Shopsys\FrameworkBundle\Model\Payment\Payment;
-use Shopsys\FrameworkBundle\Model\Payment\PaymentPriceCalculation;
-use Shopsys\FrameworkBundle\Model\Pricing\Currency\Currency;
-use Shopsys\FrameworkBundle\Model\Transport\Transport;
-use Shopsys\FrameworkBundle\Model\Transport\TransportPriceCalculation;
-use Symfony\Component\HttpFoundation\RequestStack;
-
-class TransportAndPaymentWatcher
```

Comment:
What does now control that everything is set correctly?

Reply:
If it is watcher in FE API - is it ok to have there and not here?

Reply:
This watcher was only a remnant of the previous twig implementation. The TransportAndPaymentWatcherFacade still remains.